deploy/docker-compose/configs/task-manager.json

```diff
@@ -0,0 +1,115 @@
+{
+  "WorkloadManagerDatabase": {
+    "ConnectionString": "mongodb://root:rootpassword@mongodb:27017",
+    "DatabaseName": "WorkloadManager",
+    "WorkflowCollectionName": "Workflows",
+    "WorkflowInstanceCollectionName": "WorkflowInstances",
+    "PayloadCollectionName": "Payloads",
+    "TaskDispatchEventCollectionName": "TaskDispatchEvents"
+  },
+  "WorkflowManager": {
+    "endpointSettings": {
+      "defaultPageSize": 10,
+      "maxPageSize": 1000000
+    },
+    "TaskManager": {
+      "concurrency": 1,
+      "taskTimeoutMinutes": 60,
+      "plug-ins": {
+        "argo": "Monai.Deploy.WorkflowManager.TaskManager.Argo.ArgoPlugin, Monai.Deploy.WorkflowManager.TaskManager.Argo",
```

Comment:
Do we need Argo here? this file is already in the ~/docker-compose/ scope.

Reply:
it's just a mapping used by the task manager, so doesn't really matter.

Reply:
People may look at this, get the wrong impression that Argo is also supported in this Docker Compose based version, and then define Argo task in the Workflow.